webui/src/components/Stores/BladePortStore.ts

```diff
@@ -46,6 +46,14 @@ export const useBladePortStore = defineStore('bladePort', {
                             detailsResponse.data.linkedPortUri = "NOT_FOUND";
                         }
 
+                        // Combine LinkStatus, linkWidth and linkSpeed if the port is linked up
+                        if (detailsResponse.data.linkStatus && detailsResponse.data.linkStatus == "LinkUp") {
```

Comment:
Be aware that the BMC's response is "Link Up". There is a space between "Link" and "Up"

Reply:
thank you, I have fix that